### PR TITLE
Update staff list; format better.

### PR DESCRIPTION
**Pull request by ghost**
TRM is an idiot.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,108 +2,91 @@
...
+                    </div>
+                    <div class="column">
+                        <a href="https://www.youtube.com/channel/UCi6p-LPSRkcMhEIJ2Qe0ABg/" rel="noopener noreferrer" target="_blank"><img class="someone ceo" src="/img/people/amy.jpg"></a>
+                        <h3>AtotheK</h3><span class="tag is-ceo">CEO</span>
+                    </div>
+                    <div class="column">
+                        <a href="https://dnomaid.co.uk" rel="noopener noreferrer" target="_blank"><img class="someone ceo" src="/img/people/dnomaid.jpg"></a>
+                        <h3>dnomaid</h3><span class="tag is-ceo">CEO</span>
+                    </div>
+                   	{% comment %} Make the CEOs not so big. {% endcomment %} <div class="column"></div>
```
Comment (ByteOfMelon) [inline index.html:30]: What's with the comments?

**Review by ByteOfMelon — DISMISSED** [review]

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html:30]: so people know what the bloody blank <divs> are for if they wanna edit it

Comment (ByteOfMelon) [reply index.html:30]: I don't really think that it's necessary.

Comment (ghost) [reply index.html:30]: Just accept it please dad.
Tom and others might be confused if they see that and remove it.

Comment (ByteOfMelon) [reply index.html:30]: We don't really comment the rest of our code, so...
Also, we dont need the comment 2-3 times, if it's really necessary

Comment (ghost) [reply index.html:30]: it's fiiine

**Comment by ByteOfMelon** [comment]
@ffxhand I just think it's a tad weird, since we don't comment anything else. + Espi literally just said he'd remove them lmfao

**Comment by t5r7** [comment]
Also, why are they Jekyll/Liquid comments rather than regular HTML comments?

**Pull request merged**